tests/foreman/api/test_organization.py

```diff
@@ -246,6 +246,26 @@ def test_positive_update(self, attrs):
             self.assertIn(name, new_attrs.keys())
             self.assertEqual(new_attrs[name], value)
 
+    def test_associate_user_with_organization(self):
+        """@Test: Update an organization, associate user with it.
+
+        @Assert: User is associated with organization.
+
+        @Feature: Organization
+
+        """
+        user_id = entities.User().create()['id']
+
+        client.put(
+            self.organization.path(),
+            {'organization': {'user_ids': [user_id]}},
+            verify=False,
+            auth=get_server_credentials(),
+        ).raise_for_status()
+
```

Comment:
Nitpick: drop the whitespace in this method. Whitespace is useful when it denotes significantly distinct hunks of code. This method is very small, and the whitespace needlessly makes the reader slow down and wonder what is being communicated.